lib/undiciRetry.ts

```diff
@@ -43,10 +45,9 @@ export async function sendWithRetry<T>(
 
       // success
       if (response.statusCode < 400) {
-        const resolvedBody = await resolveBody(response, retryConfig.safeParseJson)
         return {
           result: {
-            body: resolvedBody,
+            body: await resolveBody(response, retryConfig.blobBody, retryConfig.safeParseJson),
```

Comment:
why inline the resolveBody call?

Reply:
Ups, I put it here at the beginning to make a couple of tests and I forgot to change it again, I will do, sorry